src/frontend/package.json

```diff
@@ -97,6 +98,7 @@
     "@types/react-csv": "^1.1.1",
     "@types/react-virtualized-auto-sizer": "^1.0.0",
     "@types/react-window": "^1.8.2",
+    "@types/uuid": "^8.3.4",
```

Comment:
Yeah, latest `uuid` is `9.0.0`, but the latest types available is `8.3.4`. It's annoying, but the only thing we're using is the v4 uuid generator, so that part of the types is super stable b/c it's been around a super long time, so the version mismatch here is not a big deal.